docs/source/modules/event.rst

```diff
@@ -0,0 +1,51 @@
+.. _event-module:
+
+Event Module
+============
+
+.. contents::
+    :depth: 2
+
+Description
+-----------
+
+The Event module allows to create listener and assign action to event. The module is using :code:`MotechEvent` to provide
+event with specific data that will be sent by Motech Scheduler when a scheduled event is fired.
```

Comment:
MOTECH Scheduler is not the only entity that can fire events. Any module can do so, using eventRelay